psd/Export as psd.lua

```diff
@@ -441,6 +441,9 @@ function ExportToPsd(sprite, filename, frameNum)
                 else
                     -- a layer with content
                     local imageData, imageDataSize = createImageData(cel.image)
+                    
+                    -- use cel opacity if layer does not have opacity, otherwise layer opacity
+                    local opacity = layer.opacity == 255 and cel.opacity or layer.opacity
```

Comment:
```suggestion
                    -- composite layer and cel opacity
                    local opacity = math.floor((layer.opacity / 255) * cel.opacity)     
```
It seems that layer and cel opacity to be compsited (multiplied).